### PR TITLE
Remove commands integration

### PR DESCRIPTION
**Pull request by cpuguy83**
This moves all tests from `integration/commands_test.go` into an integration-cli test.

### CODE REVIEW

**Comment by LK4D4** [comment]
@cpuguy83 Doesn't compile.

**Comment by cpuguy83** [comment]
Bad rebase. One sec.

**Comment by cpuguy83** [comment]
Fixed

**Comment by LK4D4** [comment]
@cpuguy83 Test failed.

**Comment by cpuguy83** [comment]
Race condition in test, fixed.

**Comment by cpuguy83** [comment]
Argh, somethings wrong with my git-ness today.

**Comment by icecrime** [comment]
@cpuguy83 Still an issue it seems:

```
--- FAIL: TestRunAttachDetach (0.13s)
	docker_cli_run_unix_test.go:225: error executing docker inspect: exit status 1
		Error: No such image or container: attach-detach		
```

**Comment by cpuguy83** [comment]
*sigh* not sure how it even got that error.

**Comment by cpuguy83** [comment]
All good now.

File: integration/utils.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+		t.Fatal(err)
+	}
+	state, err := term.MakeRaw(pty.Fd())
+	if err != nil {
+		t.Fatal(err)
+	}
+	return state
+}
+
+func unsetRaw(t *testing.T, c *daemon.Container, state *term.State) {
```
Comment (LK4D4) [inline integration/utils.go]: Whoa, seems like this functions is not used. And we can remove awful `GetPtyMaster`!!!!

Comment (cpuguy83) [inline integration/utils.go]: Removed!

**Comment by LK4D4** [comment]
LGTM

**Comment by estesp** [comment]
LGTM

**Pull request merged**